content/_index.en.md

```diff
@@ -485,7 +485,6 @@ First of all, follow the [requirements](#requirements) above. Then, clone the re
 
 ```sh
 git clone https://github.com/OpenTermsArchive/engine.git
-cd OpenTermsArchive
```

Comment:
This should be updated: the `OpenTermsArchive` directory is not _inside_ the `OpenTermsArchive` repository, but if you `clone` from a directory, you will create a directory with the name of the repository that contains the clone, and you need to `cd` inside it 🙂 
The problem outlined in https://github.com/OpenTermsArchive/DPG-OS-CM-Open-Terms-Archive/issues/11 is that the `OpenTermsArchive` directory name is obsolete: it dates from a time when the `engine` repository was called `OpenTermsArchive`.

So, in this case, what you want is to have `cd engine` as instruction 🙂

Reply:
Yeah. I will change it then to `cd engine`